src/main/java/com/libertymutualgroup/herman/aws/ecs/loadbalancing/CertHandler.java

```diff
@@ -59,7 +59,12 @@ public SSLCertificate getSSLCertificateByUrl(String prefix, String suffix) {
     }
 
     public boolean isInternetFacingUrlScheme(SSLCertificate sslCertificate, String urlSchemeOverride) {
-        return sslCertificate.isInternetFacingUrl() && !"internal".equals(urlSchemeOverride);
-
+        boolean isInternetFacingUrlScheme = false;
+        if (!"internal".equals(urlSchemeOverride)
+                && sslCertificate != null
```

Comment:
If sslCertificate is null, we might want to throw an exception rather than silently provisioning an internal url instead

Reply:
Yeah, good thought. I'll update this.

Reply:
Updated